clubs/README.md

```diff
@@ -174,35 +190,194 @@ keep building their website to make it unique. At the second meeting, you'll
+[3]: #name-game
+[4]: #first-website-workshop
+[5]: #free-form-hacking
+[6]: #second-meeting-reminder
+
+#### Waiting for Hackers to Arrive
+
+As hackers start to arrive, your goal is to make them feel welcome. We recommend
+doing the following four things to accomplish this:
+
+1. Put the first slide of the previously mentioned presentation on the projector
+2. Put some music on (not too loud, [click here][playlist] for a good pre-made
+   Spotify playlist)
+3. Make people feel welcome as they walk in
+  - Introduce yourself
+  - Ask a couple questions to initiative conversation
+    - "How did you hear about Hack Club?"
+    - "What sorts of things are you interested in learning?"
+  - You want people to feel good about their decision to come to Hack Club
+4. Tell people what will be happening next (ex. "We're going to get the first
+   meeting started in about 10 minutes after people are done trickling in. In
+   the meantime, please make yourself at home and let me know if I can be
+   helpful with anything.")
+
+[playlist]: https://open.spotify.com/user/zachlatta/playlist/16lRCHBANauZr4utynaA2B
+
+#### Introduction Presentation
+
+The purpose of the introduction presentation is to tell members what Hack Club
+is, what you're going to be doing throughout the year, when you meet, and, _most
+importantly_, get them excited about the club.
+
+The beginning of the previously mentioned slideshow presentation contains slides
+to run with the introduction presentation.
+
+#### Name Game
+
+The name game is a simple icebreaker activity that will warm the atmosphere and
+make club members more comfortable around each other. This will be especially
+helpful to freshman, who probably don't know people from the higher grades.
+
+Have the club members gather in large circle. Start by saying your name and a
+mnemonic for it (ex. "Hey everyone, I'm Zach. Zany Zach."). Then have the person
+to your left do the same and then repeat what you said (ex. "I'm Jolly Jane. And
+he is Zany Zach"). Then have the person to their left do the same and then say
+the names and mnemonics for everyone that has gone so far, in order. Continue
+doing this through the whole circle until everyone has gone. This will get
+significantly more difficult (and fun!) as more people go -- if someone gets
+completely stuck, encourage others to help them out.
+
+#### First Website Workshop
+
+To get members started, you'll want to run the
+[Portfolio](../workshops/portfolio/README.md) workshop. It's going to walk club
+members through the process of building their first website. We recommended
+doing the following to run the Portfolio workshop (and we've prepared slides, in
+the previously linked slideshow presentation to help run it).
+
+1. Tell everyone that to start out, we're going to run the Portfolio workshop,
+   which will walk them through the process of building their first website
+2. Show the slide with the output of the Portfolio workshop and tell everyone
```

Comment:
Fuck, it does **not** take 45 minutes right now, it's like 1.5 hours I estimate because of GitHub + C9 sigup + GitHub pages signup...